arbitrum-docs/launch-orbit-chain/infra-options-orbit-chains.md

```diff
@@ -0,0 +1,71 @@
+# Infrastructure Options for Orbit Chains
+
+A successful production-grade chain often requires additional surrounding infrastructure to enable app functionality and provide a suberb user experience for those interacting with the network. There are multiple options for various components for deployers to choose from, and the below is a non-comprehensive list of providers that Orbit teams can choose to work with if the need should arise.
+
+## Chain Operation
+
+Many application developers do not have the subject matter expertise required to deploy, operate, and maintain an independent production-grade network. In these cases, we highly recommend working with one of the following RaaS providers who will manage the necessary infrastructure and ensure a high degree of performance and security for your chain:
+
+- [Caldera](https://www.caldera.xyz/)
+- [Conduit](https://conduit.xyz/)
+- [AltLayer](https://altlayer.io/)
+- [Gelato](https://www.gelato.network/)
+
+## Explorers
+
+Explorers are a basic requirement for any functional blockchain, providing a transparent view of transactions, blocks, and addresses on the blockchain. They can be accessed by users and developers to verify transactions, check the status of network activity, and ensure that transactions are being processed correctly. Some providers that can offer an explorer to Orbit developers are:
+
+- [Blockscout](https://www.blockscout.com/)
+- [Socialscan](https://socialscan.io/)
+- [Lore](https://www.lorescan.com/)
+- [Routescan](https://routescan.io/)
+
+Additionally, Orbit chains leveraging blobs for data availability may use tools like [Blobscan](https://blobscan.com/) to see which blob/block included a given transaction.
+
+
+## Bridges
+
+Orbit chains are easily launched with a canonical token bridge which permits transfers to and from their chain via Arbitrum One, Nova, or Ethereum. However, some applications may desire the ability to transfer to additional chains outside of the Orbit ecosystem or want to offer the ability to transfer across chain more quickly, without waiting for complete finality. In these instances, the following 3rd party bridging providers offer services to meet those needs:
+
+- [LayerZero](https://layerzero.network/)
+- [Connext](https://www.connext.network/)
+- [Hyperlane](https://www.hyperlane.xyz/)
+- [Axelar](https://axelar.network/)
+- [Across](https://across.to/)
+
+## Data Availability
+
+Deployers of Orbit chains may offer cheaper fees on their chain by storing the data posted in batches on an alternative Data Availability solution rather than to Ethereum directly. The built in functionality in Anytrust allows these developers to easily configure the cost of transacting and will enable more powerful features with additional chains that are also using Anytrust. However, there are integrations with Orbit that are available for those that would prefer to use one of the following third party services:
```

Comment:
```suggestion
As an Orbit chains deployer, you may offer cheaper fees on your chain by storing the data posted in batches on an alternative Data Availability solution rather than directly storing it in the parent chain. AnyTrust's built-in functionality allows you to set transaction fees easily and will enable more powerful features with additional chains that also use AnyTrust. Alternatively, you can choose from the following third-party services:
```